be/src/common/config.h

```diff
@@ -355,6 +355,7 @@ CONF_Int64(load_data_reserve_hours, "4");
 // log error log will be removed after this time
 CONF_mInt64(load_error_log_reserve_hours, "48");
 CONF_Int32(number_tablet_writer_threads, "16");
+CONF_mInt64(max_queueing_memtable_per_tablet, "2");
```

Comment:
How to determine the number? A smaller one is better, or have another knowledge to set it?
1. If the resource is full, and enlarges the flush_store number, do we need to increase this number?
2. If ingestion is real-time, the frequency is a little high, does it degreate?

Reply:
In most scenarios, there is no need to adjust this config.
if flush throughput larger than write throughput, writer will not sleep and wait, this config has no affect.
if write throughput larger than flush throughput, writer will delay 10ms at most. Based on performance test results, this has little impact on performance.